sdk/core/azure-core-http-netty/pom.xml

```diff
@@ -78,6 +78,23 @@
       <artifactId>netty-codec-http</artifactId>
     </dependency>
 
+    <dependency>
+      <groupId>io.netty</groupId>
+      <artifactId>netty-codec-http2</artifactId>
+      <version>4.1.42.Final</version>
+    </dependency>
+    <dependency>
+      <groupId>io.netty</groupId>
+      <artifactId>netty-transport-native-unix-common</artifactId>
+      <version>4.1.42.Final</version>
```

Comment:
4.1.43.Final ? Can we use this

Reply:
We are using 4.1.42.Final for all netty dependencies. So, I used the same version for these two as well. I will look into 4.1.43.Final for the next release.

Reply:
Sharing my general opinion on dependencies: we want to be fast followers rather than jumping to the newest version immediately. If we can understand where the community is (and I have tooling that can help here), we can move in step with the pack to try to minimise dependency conflicts.